examples/plugins/src/lighthouse/README.md

```diff
@@ -1,9 +1,12 @@
-# lighthouse-plugin
+# lighthouse-plugin example
 
 🕵️ **Code PushUp plugin for Lighthouse reports** 🔥
 
 ---
 
+> [ NOTE ]
```

Comment:
In order to make the nice note block, `!` has to be included. Currently it looks like a regular quote. Source[here](https://github.com/orgs/community/discussions/16925).

```suggestion
> [!NOTE]
```